prac_05/hex_colours.py

```diff
@@ -0,0 +1,18 @@
+"""
+CP1404Practical
+hexadecimal colour codes in a dictionary
+"""
+
+colour_to_code = {"Absolute Zero": "#0048ba", "Acid Green": "#b0bf1a", "AliceBlue": "#f0f8ff",
+                "Alizarin crimson": "#e32636", "Amaranth": "#e52b50", "Amber": "#ffbf00", "Amethyst": "#9966cc",
+                "Aqua": "#00ffff", "Aureolin": "#fdee00", "Brilliant Rose": "#ff55a3"}
+
+print(colour_to_code)
+
+colour_code = input("Enter colour name: ").title()
```

Comment:
The .title() operator is a good way of accounting for case sensitivity in user input. However, this doesn't work for colours, (say for example AliceBlue) that are camel case format (have no space, and are still capitalised in the middle). 
It doesn't work because it does the capitalisation on a word by word basis, which inherently requires a space.
maybe you could use the .upper() or .lower() functions instead? 😎

Reply:
I have fixed this.